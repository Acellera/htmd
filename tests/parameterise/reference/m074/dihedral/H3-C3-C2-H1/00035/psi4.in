set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 1.825465	 1.324323	 -0.951462
C	 0.788116	 1.716284	 -1.379704
C	 -0.516134	 0.957269	 -1.353318
O	 0.640172	 2.906476	 -1.947763
C	 -0.823008	 0.326187	 0.001381
H	 -1.299814	 1.647936	 -1.652547
H	 -0.470175	 0.205508	 -2.136556
C	 -0.761037	 1.342545	 1.142537
H	 -0.121016	 -0.480560	 0.187271
H	 -1.806634	 -0.131716	 -0.031989
C	 0.668504	 1.751320	 1.501944
H	 -1.329570	 2.230569	 0.870177
H	 -1.247691	 0.929214	 2.024698
C	 1.462676	 2.262592	 0.300636
H	 1.185428	 0.901239	 1.941597
H	 0.639625	 2.520894	 2.270098
H	 1.538915	 1.502354	 -0.471660
H	 2.471031	 2.544480	 0.587359
H	 0.988474	 3.134677	 -0.140853
C	 1.792514	 3.724689	 -2.034891
H	 2.180449	 3.940881	 -1.049855
H	 1.472688	 4.634305	 -2.518819
H	 2.560106	 3.239076	 -2.620117

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		15 11 8 12,
		9 5 8 12,
		1 2 3 6,
		1 2 4 20
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "23  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
