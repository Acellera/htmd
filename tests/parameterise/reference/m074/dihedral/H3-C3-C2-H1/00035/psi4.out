    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                              Psi4 (no tag) Driver

                          Git: Rev {detached?}  

    J. M. Turney, A. C. Simmonett, R. M. Parrish, E. G. Hohenstein,
    F. A. Evangelista, J. T. Fermann, B. J. Mintz, L. A. Burns, J. J. Wilke,
    M. L. Abrams, N. J. Russ, M. L. Leininger, C. L. Janssen, E. T. Seidl,
    W. D. Allen, H. F. Schaefer, R. A. King, E. F. Valeev, C. D. Sherrill,
    and T. D. Crawford, WIREs Comput. Mol. Sci. 2, 556-565 (2012)
    (doi: 10.1002/wcms.93)

                         Additional Contributions by
    A. E. DePrince, M. Saitow, U. Bozkaya, A. Yu. Sokolov
    -----------------------------------------------------------------------


    Psi4 started on: Tue Jul  5 17:38:04 2016

    Process ID:   5816
    PSI4DATADIR: /export131/home/mjharvey/miniconda3/share
    Memory level set to 256.000 MB

  ==> Input File <==

--------------------------------------------------------------------------
set {
	reference rhf
	basis 6-31G*
}


set_num_threads( 1 )
memory 2 gb

molecule MOL {
0 1

O	 1.825465	 1.324323	 -0.951462
C	 0.788116	 1.716284	 -1.379704
C	 -0.516134	 0.957269	 -1.353318
O	 0.640172	 2.906476	 -1.947763
C	 -0.823008	 0.326187	 0.001381
H	 -1.299814	 1.647936	 -1.652547
H	 -0.470175	 0.205508	 -2.136556
C	 -0.761037	 1.342545	 1.142537
H	 -0.121016	 -0.480560	 0.187271
H	 -1.806634	 -0.131716	 -0.031989
C	 0.668504	 1.751320	 1.501944
H	 -1.329570	 2.230569	 0.870177
H	 -1.247691	 0.929214	 2.024698
C	 1.462676	 2.262592	 0.300636
H	 1.185428	 0.901239	 1.941597
H	 0.639625	 2.520894	 2.270098
H	 1.538915	 1.502354	 -0.471660
H	 2.471031	 2.544480	 0.587359
H	 0.988474	 3.134677	 -0.140853
C	 1.792514	 3.724689	 -2.034891
H	 2.180449	 3.940881	 -1.049855
H	 1.472688	 4.634305	 -2.518819
H	 2.560106	 3.239076	 -2.620117

	symmetry c1
}
set optking {
	frozen_dihedral = ("
		15 11 8 12,
		9 5 8 12,
		1 2 3 6,
		1 2 4 20
	")
}

ee = energy('scf')
oeprop('DIPOLE', 'QUADRUPOLE', 'MULLIKEN_CHARGES')
f=open( 'psi4out.xyz', 'w' )
f.write( "23  " )
f.write( str(ee) + "\n" )
f.write( MOL.save_string_xyz() )
f.close()
--------------------------------------------------------------------------

  Memory set to   2.000 GiB by Python script.

*** tstart() called on cx1-1-1-4.cx1.hpc.ic.ac.uk
*** at Tue Jul  5 17:38:05 2016


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, and Andy Simmonett
                              RHF Reference
                        1 Threads,   2000 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c1
    Full point group: C1

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           O          1.229059752635    -0.517888828897    -0.405783131625    15.994914619560
           C          0.191710752635    -0.125927828897    -0.834025131625    12.000000000000
           C         -1.112539247365    -0.884942828897    -0.807639131625    12.000000000000
           O          0.043766752635     1.064264171103    -1.402084131625    15.994914619560
           C         -1.419413247365    -1.516024828897     0.547059868375    12.000000000000
           H         -1.896219247365    -0.194275828897    -1.106868131625     1.007825032070
           H         -1.066580247365    -1.636703828897    -1.590877131625     1.007825032070
           C         -1.357442247365    -0.499666828897     1.688215868375    12.000000000000
           H         -0.717421247365    -2.322771828897     0.732949868375     1.007825032070
           H         -2.403039247365    -1.973927828897     0.513689868375     1.007825032070
           C          0.072098752635    -0.090891828897     2.047622868375    12.000000000000
           H         -1.925975247365     0.388357171103     1.415855868375     1.007825032070
           H         -1.844096247365    -0.912997828897     2.570376868375     1.007825032070
           C          0.866270752635     0.420380171103     0.846314868375    12.000000000000
           H          0.589022752635    -0.940972828897     2.487275868375     1.007825032070
           H          0.043219752635     0.678682171103     2.815776868375     1.007825032070
           H          0.942509752635    -0.339857828897     0.074018868375     1.007825032070
           H          1.874625752635     0.702268171103     1.133037868375     1.007825032070
           H          0.392068752635     1.292465171103     0.404825868375     1.007825032070
           C          1.196108752635     1.882477171103    -1.489212131625    12.000000000000
           H          1.584043752635     2.098669171103    -0.504176131625     1.007825032070
           H          0.876282752635     2.792093171103    -1.973140131625     1.007825032070
           H          1.963700752635     1.396864171103    -2.074438131625     1.007825032070

  Running in c1 symmetry.

  Rotational constants: A =      0.09293  B =      0.04647  C =      0.03836 [cm^-1]
  Rotational constants: A =   2785.92499  B =   1393.17571  C =   1150.11917 [MHz]
  Nuclear repulsion =  536.728586645712880

  Charge       = 0
  Multiplicity = 1
  Electrons    = 72
  Nalpha       = 36
  Nbeta        = 36

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is CORE.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G*
    Number of shells: 82
    Number of basis function: 163
    Number of Cartesian functions: 163
    Spherical Harmonics?: false
    Max angular momentum: 2

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A        163     163       0       0       0       0
   -------------------------------------------------------
    Total     163     163      36      36      36       0
   -------------------------------------------------------

 OEINTS: Overlap, kinetic, potential, dipole, and quadrupole integrals
         stored in file 35.

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              1
    Integrals threads:           1
    Memory (MB):              1430
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: 
    Number of shells: 342
    Number of basis function: 1079
    Number of Cartesian functions: 1079
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 1.7322985332E-03.
  Using Symmetric Orthogonalization.
  SCF Guess: Core (One-Electron) Hamiltonian.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   1:  -259.34257675530489   -2.59343e+02   8.26929e-02 
   @DF-RHF iter   2:  -194.87733500435627    6.44652e+01   6.27440e-02 DIIS
   @DF-RHF iter   3:  -321.39749328163464   -1.26520e+02   5.91505e-02 DIIS
   @DF-RHF iter   4:  -392.54450193368916   -7.11470e+01   3.91070e-02 DIIS
   @DF-RHF iter   5:  -404.21296574813061   -1.16685e+01   2.81163e-02 DIIS
   @DF-RHF iter   6:  -420.92336223561620   -1.67104e+01   8.34181e-03 DIIS
   @DF-RHF iter   7:  -421.77970151661066   -8.56339e-01   3.59680e-03 DIIS
   @DF-RHF iter   8:  -421.90981285786870   -1.30111e-01   1.34497e-03 DIIS
   @DF-RHF iter   9:  -421.94033206345739   -3.05192e-02   4.19990e-04 DIIS
   @DF-RHF iter  10:  -421.94397864064752   -3.64658e-03   1.73133e-04 DIIS
   @DF-RHF iter  11:  -421.94566649250612   -1.68785e-03   1.42834e-04 DIIS
   @DF-RHF iter  12:  -421.94662292483758   -9.56432e-04   1.04095e-04 DIIS
   @DF-RHF iter  13:  -421.94752465667489   -9.01732e-04   4.85406e-05 DIIS
   @DF-RHF iter  14:  -421.94777501719773   -2.50361e-04   1.11347e-05 DIIS
   @DF-RHF iter  15:  -421.94778630058033   -1.12834e-05   4.16678e-06 DIIS
   @DF-RHF iter  16:  -421.94778734250224   -1.04192e-06   1.05011e-06 DIIS
   @DF-RHF iter  17:  -421.94778739284322   -5.03410e-08   3.39297e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A    -20.582687     2A    -20.553341     3A    -11.333146  
       4A    -11.274552     5A    -11.233796     6A    -11.221616  
       7A    -11.215323     8A    -11.214264     9A    -11.210501  
      10A     -1.769387    11A     -1.420291    12A     -1.089912  
      13A     -1.054909    14A     -0.976133    15A     -0.954290  
      16A     -0.907511    17A     -0.839105    18A     -0.810831  
      19A     -0.771987    20A     -0.719976    21A     -0.694443  
      22A     -0.660610    23A     -0.624258    24A     -0.620040  
      25A     -0.596241    26A     -0.582733    27A     -0.551950  
      28A     -0.538942    29A     -0.522415    30A     -0.500251  
      31A     -0.481550    32A     -0.474319    33A     -0.459347  
      34A     -0.452275    35A     -0.447632    36A     -0.178676  

    Virtual:                                                              

      37A      0.139619    38A      0.233262    39A      0.262978  
      40A      0.275379    41A      0.284662    42A      0.296789  
      43A      0.313858    44A      0.316685    45A      0.317130  
      46A      0.324354    47A      0.332833    48A      0.345428  
      49A      0.367414    50A      0.370159    51A      0.389828  
      52A      0.399288    53A      0.436189    54A      0.458476  
      55A      0.465851    56A      0.471391    57A      0.522570  
      58A      0.566793    59A      0.640881    60A      0.711078  
      61A      0.722578    62A      0.744733    63A      0.752098  
      64A      0.756696    65A      0.772853    66A      0.792007  
      67A      0.847663    68A      0.864714    69A      0.871912  
      70A      0.898712    71A      0.914829    72A      0.926655  
      73A      0.934205    74A      0.968532    75A      1.004617  
      76A      1.030238    77A      1.055546    78A      1.080140  
      79A      1.105767    80A      1.118163    81A      1.133648  
      82A      1.149491    83A      1.158721    84A      1.164914  
      85A      1.165879    86A      1.186612    87A      1.207871  
      88A      1.213099    89A      1.231710    90A      1.238994  
      91A      1.261243    92A      1.262673    93A      1.272201  
      94A      1.302023    95A      1.307580    96A      1.334990  
      97A      1.403296    98A      1.444776    99A      1.466774  
     100A      1.491701   101A      1.552178   102A      1.589225  
     103A      1.627091   104A      1.670223   105A      1.733157  
     106A      1.754216   107A      1.795894   108A      1.819937  
     109A      1.880369   110A      1.894576   111A      1.933938  
     112A      1.993387   113A      2.018596   114A      2.030339  
     115A      2.087695   116A      2.108055   117A      2.122730  
     118A      2.178125   119A      2.215588   120A      2.239595  
     121A      2.256435   122A      2.269751   123A      2.314909  
     124A      2.373452   125A      2.388046   126A      2.404882  
     127A      2.411805   128A      2.437461   129A      2.474969  
     130A      2.525196   131A      2.540780   132A      2.593767  
     133A      2.628942   134A      2.648055   135A      2.672315  
     136A      2.686707   137A      2.693427   138A      2.733591  
     139A      2.758321   140A      2.791094   141A      2.822070  
     142A      2.847173   143A      2.902539   144A      2.968905  
     145A      3.003928   146A      3.019364   147A      3.057060  
     148A      3.093579   149A      3.162870   150A      3.229297  
     151A      3.345221   152A      3.453951   153A      3.554613  
     154A      4.476428   155A      4.583581   156A      4.599030  
     157A      4.680537   158A      4.775997   159A      4.799634  
     160A      4.955658   161A      5.021173   162A      5.061340  
     163A      5.298756  

    Final Occupation by Irrep:
              A 
    DOCC [    36 ]

  Energy converged.

  @DF-RHF Final Energy:  -421.94778739284322

   => Energetics <=

    Nuclear Repulsion Energy =            536.7285866457128805
    One-Electron Energy =               -1638.2741947228373647
    Two-Electron Energy =                 679.5978206842800091
    DFT Exchange-Correlation Energy =       0.0000000000000000
    Empirical Dispersion Energy =           0.0000000000000000
    PCM Polarization Energy =               0.0000000000000000
    EFP Energy =                            0.0000000000000000
    Total Energy =                       -421.9477873928445888

    Alert: EFP and PCM quantities not currently incorporated into SCF psivars.

Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr
  ==> Properties <==


Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -1.4825      Y:     0.9662      Z:     4.5833

  Electronic Dipole Moment: (a.u.)
     X:     1.6864      Y:    -0.8127      Z:    -4.6452

  Dipole Moment: (a.u.)
     X:     0.2039      Y:     0.1535      Z:    -0.0619     Total:     0.2626

  Dipole Moment: (Debye)
     X:     0.5183      Y:     0.3902      Z:    -0.1574     Total:     0.6676


  Saving occupied orbitals to File 180.

*** tstop() called on cx1-1-1-4.cx1.hpc.ic.ac.uk at Tue Jul  5 17:38:32 2016
Module time:
	user time   =      26.80 seconds =       0.45 minutes
	system time =       0.26 seconds =       0.00 minutes
	total time  =         27 seconds =       0.45 minutes
Total time:
	user time   =      26.80 seconds =       0.45 minutes
	system time =       0.26 seconds =       0.00 minutes
	total time  =         27 seconds =       0.45 minutes


Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the  density matrix

  Nuclear Dipole Moment: (a.u.)
     X:    -1.4825      Y:     0.9662      Z:     4.5833

  Electronic Dipole Moment: (a.u.)
     X:     1.6864      Y:    -0.8127      Z:    -4.6452

  Dipole Moment: (a.u.)
     X:     0.2039      Y:     0.1535      Z:    -0.0619     Total:     0.2626

  Dipole Moment: (Debye)
     X:     0.5183      Y:     0.3902      Z:    -0.1574     Total:     0.6676

  Quadrupole Moment: (Debye Ang)
    XX:   -56.0382     YY:   -55.4796     ZZ:   -57.2829
    XY:     3.5513     XZ:    -1.5093     YZ:    -1.3310

  Traceless Quadrupole Moment: (Debye Ang)
    XX:     0.2287     YY:     0.7873     ZZ:    -1.0160
    XY:     3.5513     XZ:    -1.5093     YZ:    -1.3310

  Mulliken Charges: (a.u.)
   Center  Symbol    Alpha    Beta     Spin     Total
       1     O     4.09164  4.09164  0.00000 -0.18329
       2     C     2.78936  2.78936  0.00000  0.42128
       3     C     3.16533  3.16533  0.00000 -0.33067
       4     O     4.30301  4.30301  0.00000 -0.60602
       5     C     3.16535  3.16535  0.00000 -0.33070
       6     H     0.40009  0.40009  0.00000  0.19982
       7     H     0.42230  0.42230  0.00000  0.15540
       8     C     3.15201  3.15201  0.00000 -0.30403
       9     H     0.41954  0.41954  0.00000  0.16093
      10     H     0.41554  0.41554  0.00000  0.16891
      11     C     3.13434  3.13434  0.00000 -0.26869
      12     H     0.42007  0.42007  0.00000  0.15986
      13     H     0.42049  0.42049  0.00000  0.15903
      14     C     3.30487  3.30487  0.00000 -0.60975
      15     H     0.42468  0.42468  0.00000  0.15064
      16     H     0.41860  0.41860  0.00000  0.16279
      17     H     0.41799  0.41799  0.00000  0.16402
      18     H     0.40631  0.40631  0.00000  0.18739
      19     H     0.39891  0.39891  0.00000  0.20218
      20     C     3.10264  3.10264  0.00000 -0.20527
      21     H     0.40731  0.40731  0.00000  0.18538
      22     H     0.40440  0.40440  0.00000  0.19120
      23     H     0.41520  0.41520  0.00000  0.16959

   Total alpha = 36.00000, Total beta = 36.00000, Total charge =  0.00000


*** PSI4 exiting successfully. Buy a developer a beer!
